source/error.c

```diff
@@ -150,9 +150,8 @@ void aws_register_error_info(const struct aws_error_info_list *error_info) {
     AWS_ASSERT(error_info->error_list);
     AWS_ASSERT(error_info->count);
 
-    int min_range = error_info->error_list[0].error_code;
-
-    int slot_index = min_range >> SLOT_DIV_SHIFT;
+    const int min_range = error_info->error_list[0].error_code;
+    const int slot_index = min_range >> SLOT_DIV_SHIFT;
 
     AWS_ASSERT(slot_index < AWS_MAX_ERROR_SLOTS && slot_index >= 0);
```

Comment:
If this is in prod mode, we'll return random data here. Should this either be a fatal assert, or a check that slot_index is in range, followed by returning a "this was a bad slot" error message?

Reply:
I can definitely make it a fatal assert. I think this code predates that macro.

Reply:
WAit no, it's fatal asserted right below. I can remove this check.